src/Console/ReplayCommand.php

```diff
@@ -45,7 +46,7 @@ public function handle(Projectionist $projectionist): void
     public function selectProjectors(array $projectorClassNames): ?Collection
     {
         if (count($projectorClassNames) === 0) {
-            if (! $this->confirm('Are you sure you want to replay events to all projectors?', true)) {
+            if ($this->isRunningInteractively() && ! $this->confirm('Are you sure you want to replay events to all projectors?', true)) {
```

Comment:
```suggestion
            if (! $this->option('force') && ! $this->confirm('Are you sure you want to replay events to all projectors?', true)) {
```

I don't think this is worth extracting to a method.